### ISSUE TITLE
Add support for NET9.0 in Visual Studio 2022 image

### ISSUE THREAD

**Issue by jrgcubano** [appveyor/ci#3936]
NET9 has been officially released and it would be nice to have it available to start using it with our builds and deploys.

Thanks in advance.

**Comment by jrgcubano** [comment]
@OwenMcDonnell It would be appreciated if it could also be included in the Ubuntu images. Thanks

**Comment by Jaxelr** [comment]
FYI, as a workaround I usually do this:

```yaml
install:
- ps: |
    Invoke-WebRequest -Uri 'https://dot.net/v1/dotnet-install.ps1' -UseBasicParsing -OutFile "$env:temp\dotnet-install.ps1"
    & $env:temp\dotnet-install.ps1 -Architecture x64 -Version '9.0.100' -InstallDir "$env:ProgramFiles\dotnet"
before_build:
  - cmd: dotnet --version
```

Might be unsuitable for your use case, but it might unblock you for the short term :)

**Comment by jrgcubano** [comment]
@OwenMcDonnell Hello.

You have already deployed the new image with NET9, right? The deployment includes a new version of git that has issues and Appveyor cannot clone any repository.

![image](https://github.com/user-attachments/assets/0b89f01b-e320-468b-848e-85c9350ff436). 

More info here https://stackoverflow.com/questions/79074800/ue-git-cloning-stuck-at-0

We have had to start using the 'Previous Visual Studio 2022' image that uses the previous git version (2.46.0.windows.1) until this issue is fixed.

I think the problem has been fixed in version 2.47.1.windows.1.

Thanks in advance.

**Comment by OwenMcDonnell** [comment]
@jrgcubano Yes, the new image was deployed but then rolled back to address this issue. If it really is just a problem with git 2.47.0 then the image should be re-deployed pretty soon. But we'll have to verify. 
You can continue to use `Visual Studio 2022` image as that is now the previous and once we roll out a new image again, we'll be monitoring support channels for re-occurrence of any issues.

**Comment by grantcolley** [comment]
What is the ETA for the new image of Visual Studio 2022 supporting NET9?

**Comment by OwenMcDonnell** [comment]
Sorry for the late response. Images with dotnet 9 sdk are live thought still undocumented. We've run in to a few more difficult problems than usual this time around. We will be re-releasing and updated image that addresses those issues in the next few days, and documentation following shortly after.

**Comment by OwenMcDonnell** [comment]
https://www.appveyor.com/updates/
